pkg/routes/openidmetadata.go

```diff
@@ -67,7 +67,7 @@ func (s *OpenIDMetadataServer) Install(c *restful.Container) {
 	cfg.Path(serviceaccount.OpenIDConfigPath).Route(
 		cfg.GET("").
 			To(fromStandard(s.serveConfiguration)).
-			Doc("get service account issuer OpenID configuration, also known as the 'OIDC discovery doc'").
+			Doc("get service account issuer OpenID configuration, also known as the 'OIDC discovery doc'. Beta.").
```

Comment:
Suggest something like "This is a beta endpoint that requires the feature gate ServiceAccountIssuerDiscovery to be enabled."

Reply:
Done.